requirements.json

```diff
@@ -22,7 +22,7 @@
   "cygwin.exe": {
     "bundle": "yes",
     "url": "https://cygwin.com/setup-x86_64.exe",
-    "sha256sum": "08079a13888b74f6466def307a687e02cb26fc257ea2fa78d40f02e28330fd56"
+    "sha256sum": "58f9f42f5dbd52c5e3ecd24e537603ee8897ea15176b7acdc34afcef83e5c19a"
```

Comment:
this is windows app and windows sha256 should be used, but it is interesting why. It uses Cryptographic hash function to calculate hash string and if message (.exe file) is the same, the message digest should also be the same.